projectq/backends/_circuits/_to_latex.py

```diff
@@ -370,6 +382,72 @@ def _gate_name(self, gate):
             name = str(gate)
         return name
 
+    def _sqswap_gate(self, lines, ctrl_lines, daggered):
+        """
+        Return the TikZ code for a Square-root Swap-gate.
+
+        Args:
+            lines (list<int>): List of length 2 denoting the target qubit of
+                the Swap gate.
+            ctrl_lines (list<int>): List of qubit lines which act as controls.
+            daggered (bool): Show the daggered one if True.
+        """
+        assert(len(lines) == 2)  # sqrt swap gate acts on 2 qubits
+        delta_pos = self._gate_offset(SqrtSwap)
+        gate_width = self._gate_width(SqrtSwap)
+        lines.sort()
+
+        gate_str = ""
+        for line in lines:
+            op = self._op(line)
+            w = "{}cm".format(.5 * gate_width)
+            s1 = "[xshift=-{w},yshift=-{w}]{op}.center".format(w=w, op=op)
+            s2 = "[xshift={w},yshift={w}]{op}.center".format(w=w, op=op)
+            s3 = "[xshift=-{w},yshift={w}]{op}.center".format(w=w, op=op)
+            s4 = "[xshift={w},yshift=-{w}]{op}.center".format(w=w, op=op)
+            swap_style = "swapstyle,edgestyle"
+            if self.settings['gate_shadow']:
+                swap_style += ",shadowed"
+            gate_str += ("\n\\node[swapstyle] ({op}) at ({pos},-{line}) {{}};"
+                         "\n\\draw[{swap_style}] ({s1})--({s2});\n"
+                         "\\draw[{swap_style}] ({s3})--({s4});"
+                         ).format(op=op, s1=s1, s2=s2, s3=s3, s4=s4,
+                                  line=line, pos=self.pos[line],
+                                  swap_style=swap_style)
+
+        # add a circled 1/2
+        midpoint = (lines[0] + lines[1]) / 2.
+        pos = self.pos[lines[0]]
+        op_mid = "line{}_gate{}".format(
+            '{}-{}'.format(*lines), self.op_count[lines[0]])
+        gate_str += ("\n\\node[xstyle] ({op}) at ({pos},-{line})\
+                {{\\scriptsize $\\frac{{1}}{{2}}{dagger}$}};"
+                     ).format(op=op_mid, line=midpoint, pos=pos,
+                              dagger='^{{\dagger}}' if daggered else '')
+
+        # add two vertical lines to connect circled 1/2
+        gate_str += "\n\\draw ({}) edge[edgestyle] ({});".format(
+            self._op(lines[0]), op_mid)
+        gate_str += "\n\\draw ({}) edge[edgestyle] ({});".format(
+            op_mid, self._op(lines[1]))
+
+        if len(ctrl_lines) > 0:
```

Comment:
This part of the code gets never executed in the tests. Would be great to write a test covering this in order to keep our almost 100% test coverage :)